docs/openapi3.yaml

```diff
@@ -1856,7 +1996,8 @@ components:
             The event body will include detailed information about the test, such as the number of scenarios that were executed and the number of requests that were invoked.
           items:
             type: string
-            description: The url of where to send the webhook with the report information
+            format: uuid
```

Comment:
there is two use cases for webhooks
1. ad hoc webhook per specific test (what we have today)
2. global webhooks for all predator tests (what we want to add)

regarding 1:
we can keep the current behavior, maybe add to the job resource also param which defines the 'format' 
or maybe we can change it to use ids (although it's breaking change, although it's mainly internal use) and create ad-hoc webhook which will be one time (won't be returned at get all results)
if we do it need add index on this column (one_time)

Reply:
I wanted to allow the user to specify what webhooks he would like to attach to that job(that is a breaking change, true, but not hard to migrate),
maybe webhooks resource should have `global` property, which will define if want to apply them globally.

Reply:
I agree that we should change this to use `UUID`s because URL, although keeps same functionality without breaking it, is just another way to use webhooks and we should merge the webhooks feature into only one way of use, and not have 2 different ones because of "legacy" code

I agree with the `global` property. As i see it, when users createsa webhook, they can either mark it as `global` and it will take into effect in all future test runs, or they can assign it to a specific test run (`Job`) - as is done here with a list of `UUID`s 

I don't have a problem with past webhooks functionality in versions 1.3 and below to be broken on 1.4 and anyone upgrading to 1.4 has to update their webhooks - once it'll be available in the UI it should be a matter of minutes and much more flexible than it is now